tckprocess.adoc

```diff
@@ -0,0 +1,406 @@
+# MicroProfile TCK Process 1.0
+
+Specification Teams under the Eclipse Foundation Specification
+Process MUST produce a Technology Compatibility Kit (TCK) that delivers
+on the promise of enabling multiple compatible implementations.
+
+This document defines:
+
+-   Materials a TCK MUST possess to be considered suitable for
+    delivering portability
+
+-   Process for challenging tests and how these challenges are resolved
+
+-   Means of excluding released TCK tests from certification
+    requirements
+
+-   Policy on improving TCK tests for released specifications
+
+-   Process for self-certification
+
+[#_materials_for_a_tck_release]
+## Materials for a TCK Release
+
+Artifacts:
+
+-   Specification contributors MUST produce the team licensed (EPL, Apache, etc.)
+    version for distribution to Maven Central or other open source
+    channels, usable for automated, non-official testing, and
+    implementation. No compatibility claims can be made on the basis of
+    this TCK.
+
+-   Specification contributors MUST produce a final candidate binary that includes the
+    https://www.eclipse.org/legal/tck.php[EFTL] license. The MicroProfile
+    Specification Committee will sign and promote the Specification's TCK binary
+    for distribution via Eclipse infrastructure on final approval. This
+    is the TCK binary usable for self-certification when one desires to
+    make a claim of compatibility.
+
+-   Both TCK binaries MUST contain the following
+
+    -   User guide outlining
+
+        -   Software requirements
+
+        -   Installation and configuration
+
+        -   How to run the tests
+
+        -   Where to file challenges
+
+        -   TCK specific rules not covered in this process guide
+
+    -   Instructions describing how to run the compatible
+        implementation(s) that are being used to validate the TCK
```

Comment:
It seems out of place for the TCK binary to contain details of particular vendor implementations. Can that information be provided separately from the TCK binary?

Reply:
It is about the README on how to run the tck must be present. Not sure why you think it is out of place @njr-11

Reply:
A readme on how to run the TCK sounds good, including details such as what @xstefank mentions above. I like that interpretation better. But that isn't what I understood from, "Instructions describing how to run the compatible implementation(s)".  Maybe it should be changed to, "Instructions describing how to run the TCK that can be followed by compatible implementation(s)..." ?